zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/ElasticsearchSpanConsumer.java

```diff
@@ -47,19 +48,26 @@
   final ElasticsearchStorage es;
   final Set<String> autocompleteKeys;
   final IndexNameFormatter indexNameFormatter;
+  final char indexTypeDelimiter;
   final boolean searchEnabled;
   final DelayLimiter<AutocompleteContext> delayLimiter;
 
   ElasticsearchSpanConsumer(ElasticsearchStorage es) {
     this.es = es;
     this.autocompleteKeys = new LinkedHashSet<>(es.autocompleteKeys());
     this.indexNameFormatter = es.indexNameFormatter();
+    this.indexTypeDelimiter = es.indexTypeDelimiter();
```

Comment:
super note: literally this is not enough! let's please defocus from this being the single issue blocking ES 7 support